common-arcana.lic

```diff
@@ -341,4 +348,12 @@ module DRCA
 
     cast?(data['cast'], data['symbiosis'], data['before'], data['after'])
   end
+
+  def check_to_harness(should_harness)
+    return false unless should_harness.equals? 'true'
```

Comment:
Why not `return false unless should_harness`?

Reply:
Thought about this, but I figured maybe someone would want to turn it off and on, and if they do so in their yamls, they'll have to remember that this works with removal and not with false, so they can swap false and true comfortably without having to know how the internals of Ruby work. It seems more user friendly with this, if that makes sense.

Reply:
I actually worked it the way you are suggesting, but like I said, this is more user friendly. I think that when left on, this setup can actually hurt someone pretty hard without the attunement to support it.

Reply:
I'm not sure I understand. The simplified version should work for both true and false (and nil).

Reply:
Hm, then maybe I am not understanding.

If I put should_harness, doesn't that check, like our flags, if something is there or not?
It's just checking if it has a string?

Reply:
There's no outright boolean value, so it's saying "if it's got something there, it's true"

Reply:
OK, thanks for that stuff, got it sorted.